docs/getting-started.md

```diff
@@ -133,47 +131,46 @@ EXPOSE 80
 ENTRYPOINT ["npm","run","start"]
 ```
 
-*You can find all the different `Dockerfile` samples for all languages [here](docker-examples.md).*
+_You can find all the different `Dockerfile` samples for all languages [here](docker-examples.md)._
 
 The interesting part is the last line, you need to update the entry point depending on the project you are using
 
 the only rule that endpoint must
-- expose the app on port `80`
-- if the hostname is required by the framework/lib then it has to be set to `0.0.0.0`
 
-For our example `npm run prod` is running `PORT=80 node index.js` which satisfies these two requirements
+* expose the app on port `80`
+* if the hostname is required by the framework/lib then it has to be set to `0.0.0.0`
 
+For our example `npm run prod` is running `PORT=80 node index.js` which satisfies these two requirements
 
 ## Create a Kintoblock on KintoHub from the service we just created
 
-*For more details and screenshots about creating KintoBlocks read more [here](creating-a-kintoblock.md)*
+_For more details and screenshots about creating KintoBlocks read more [here](creating-a-kintoblock.md)_
 
 By this point, you should already have a GitHub repository that is compatible with KintoHub
 
 Next step is connecting your workspace to your GitHub account. Go to edit workspace and click on connect GitHub button.
 
-Now go to create Kintoblock page. Add a Kintoblock name *(name must be unique across all accounts)*, choose an existing repository and select the repository you just created. Set protocol to `HTTP` and language to `Node.js`.
+Now go to create Kintoblock page. Add a Kintoblock name _(name must be unique across all accounts)_, choose an existing repository and select the repository you just created. Set protocol to `HTTP` and language to `Node.js`.
```

Comment:
suggest adding all requirements to KB (and KA) creation name here: unique across all accounts, all small caps, no spaces, etc

Reply:
Do you have a list of the requirements? Just to make sure i dont leave something out? 💃

Reply:
@nadeemkhedr has it - not sure if the excel files have been updated along the last changes

Reply:
This is on the dashboard at the moment so I took it from there "name must be unique across all accounts, only lowercase characters and digits are allowed - no spaces and no caps for now please"